core/templates/domain/exploration/ParamSpecObjectFactory.ts

```diff
@@ -28,25 +28,17 @@ export interface IParamSpecBackendDict {
 }
 
 export class ParamSpec {
-  _objType: ParamType;
-  /**
-   * @constructor
-   * @param {!ParamType} objType - The type of the parameter.
-   */
-  constructor(objType: ParamType) {
-    /** @member {ParamType} */
-    this._objType = objType;
-  }
+  constructor(private objType: ParamType) {}
```

Comment:
Because all of the information the comment provided is now in the type, so it's now redundant (and likely to become out-of-date), so I found it better to just remove it.

Reply:
I see. In that case do you think we should remove comment in other functions as well? I'm fine with whatever pattern we follow in other files.

Reply:
Done, I've left JSdocs that leave valuable descriptions of arguments alone though.